data/brands/amenity/restaurant.json

```
@@ -193,6 +193,17 @@
         "official_name": "Bahama Breeze Island Grille"
       }
     },
+    {
+      "displayName": "Baguette & Baguette",
+      "id": "eatnpark-96af40",
+      "locationSet": {"include": ["tn"]},
+      "tags": {
+        "amenity": "fast_food",
```

```suggestion
        "amenity": "fast_food",
```
This should probably go in `fast_food.json` instead